documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+   3. The name must end with the year of the last day of the competition.
+   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, and 0-9, with or without accents. Only '.:-& are permitted as special characters.
+   5. The competition name must be 50 characters or less, including whitespaces.
+   6. If the competition is named after anyone or anything, they must be persons or things lacking any controversial meaning worldwide.
+   7. Names should contain the location of the competition.
+   8. Competitions may not be named after a living cuber, competition organizer, or any competition staff.
+   9. Names must not contain possibly offensive parts or memes.
+   10. The name cannot contain website domains or anything susceptible to be interpreted as a link, hashtag or in general an electronic identifier.
+   11. The name cannot contain any trademarks (such as "Nike") unless the organization team is formally authorized, in writing, to use such names.
+   12. Names should not contain any unnecessary parts in the name, like "competition".
+   13. The competition nickname and ID must resemble the competition name.
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
+      1. Otherwise, the WCA Board should review the circumstances of the proposed competition. A Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+         1. The two competitions don’t have events in common.
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
```

Comment:
I believe around here we should add something about the deadline for adding a venue/location to the Fewest Moves Simultaenous competitions. WCAT already mentioned in the Crash-course that "The latest deadline to add a new location is 28 days before the competition."

Reply:
Sure, that's reasonable.

Reply:
If you feel like suggesting a change directly, that would be appreciated! :D

Reply:
I did slightly above. Feel free to change wording as seen fit.

Reply:
Thanks! I'm resolving this conversation in favor of the new one :smile: